lib/super_diff/rspec.rb

```diff
@@ -62,6 +62,11 @@ def self.fuzzy_object?(value)
     end
 
     SuperDiff.configuration.tap do |config|
+      config.set_alpha_color(:magenta)
```

Comment:
Since colors is a global SuperDiff configuration and not just specific to the RSpec integration, what do you think about setting default values for these in Configuration#initialize?